README.md

```diff
@@ -8,6 +8,73 @@ so it can directly be executed as a native application on the target platform.
 [![Travis CI](https://api.travis-ci.org/gluonhq/client-maven-plugin.svg?branch=master)](https://travis-ci.org/gluonhq/client-maven-plugin)
 [![BSD-3 license](https://img.shields.io/badge/license-BSD--3-%230778B9.svg)](https://opensource.org/licenses/BSD-3-Clause)
 
+### Requirements
+
+#### Mac OS X and iOS
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-darwin-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-darwin-20.1.0-ea+28
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+##### Additional requirements
+
+* iOS can be built only on Mac OS X
+
+* Install `Homebrew`, if you haven't already. Please refer to https://brew.sh/ for more information.
+
+* Install `libusbmuxd`
+
+Using `brew`:
+
+    brew install --HEAD libusbmuxd
+
+* Install `libimobiledevice`
+
+Using `brew`:
+
+    brew install --HEAD libimobiledevice
+
+#### Linux and Android
+
+* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-linux-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+
+* Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
+
+For example:
+
+    export GRAALVM_HOME=/opt/graalvm-svm-linux-20.1.0-ea+28
+
+* Set `JAVA_HOME` to point to the GraalVM installation directory
+
+For example:
+
+    export JAVA_HOME=$GRAALVM_HOME
+
+##### Additional requirements
+
+* Android can be built only on Linux OS
+
+The client plugin will download the Android SDK and install the required packages. 
+
+Alternatively, you can define a custom location to the Android SDK by setting the `ANDROID_SDK` environment variable, making sure that you have installed all the packages from the following list:
+
+* platforms;android-27
+* platform-tools
+* build-tools;27.0.3
+* extras;android;m2repository
+* extras;google;m2repository
+* ndk-bundle (in case it is not installed separately, then set the `ANDROID_NDK` environment variable accordingly)
```

Comment:
May be update this to:

> User can opt to skip this bundle and download Android NDK package separately. They will then have to set `ANDROID_NDK` environment variable to its location.

Reply:
ok, updated

Reply:
@abhinayagarwal can you check?

Reply:
Looks good

Reply:
Maybe you should provide an example because setting ANDROID_NDK can be tricky. E.g., I have to set it like this:

    export ANDROID_SDK=~/Library/Android/sdk/
    export ANDROID_NDK=~/Library/Android/sdk/ndk/21.1.6352462/

If you just set it to the ndk folder, like the SDK, it will not work.

Reply:
@abhinayagarwal Cool, so then this PR is ready, or am I missing something?

@mipastgt Hmm, not sure if we should get into such detail.